File: Amethyst/SiteAnalytics.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Amethyst
{
    public partial class SiteAnalytics : UserControl
    {
        public SiteAnalytics()
        {
            InitializeComponent();
        }

        private void tmData_Tick(object sender, EventArgs e)
        {
            lblSiteName.Text = "Network Name: " + Properties.Settings.Default.SiteName;
            lblAdIntensity.Text = "Advertising Intensity: " + Properties.Settings.Default.AdIntensity.ToString();
            lblUserCount.Text = "Users: " + Properties.Settings.Default.Users.ToString();
            lblAdsense.Text = "AdSense? : " + UpgradeMeta.checkUpgradeBought("Enable AdSense");
        }
    }
}

File: vb-archive/Amethyst/SiteAnalytics.vb
Public Class SiteAnalytics
    Private Sub tmData_Tick(sender As Object, e As EventArgs) Handles tmData.Tick
        lblSiteName.Text = "Network Name: " & My.Settings.SiteName
        lblAdIntensity.Text = "Advertising Intensity: " & My.Settings.AdIntensity.ToString()
        lblUserCount.Text = "Users: " & My.Settings.Users.ToString()
    End Sub
End Class